src/store/storeSocket.ts

```diff
@@ -2,13 +2,18 @@ import { throttle } from 'lodash';
 import store from './store';
 import { logger } from '../logger';
 import { Server as SocketIoServer, Namespace as SocketIoNamespace } from 'socket.io';
+import { info } from 'console';
 
 let previousData;
 function sendDataChanges(io: SocketIoServer | SocketIoNamespace) {
 	const currentData = store.getState().data;
 
+	io.to(`/fleet`).emit('fleetUpdate');
```

Comment:
Is this needed? 🤔

Reply:
Yes, is needed. Without this survivor count won't update on infoboard when there are changes in the fleet data and survivor count. Tested.